### DISCUSSION TITLE
tabwrangler not close tab

### DISCUSSION THREAD

**Discussion by vfi2024** [tabwrangler/tabwrangler#467] [General]
I settings close inactive tabs to 2 min but tabs not close after 2 min

**Comment by ssorallen** [comment]
@vfi2024  Could you see if Tab Wrangler is throwing any exceptions?

1. Go to "Window" > "Extensions" in Chrome
2. Toggle "Developer Mode" in upper right corner
3. Click the "service worker" link that says Inspect views service worker next to Tab Wrangler
4. Paste any red exceptions from the Console into this issue

Thank you.

**Comment by ssorallen** [comment]
This issue is being tracked in #466.
